### PR TITLE
ENH: Add capm to financial.py

### PR DESCRIPTION
**Pull request by olamilekan000**
I just added capm to financial.py

### CODE REVIEW

File: numpy/lib/financial.py [hunk 1]
```diff
@@ -759,3 +759,51 @@ def mirr(values, finance_rate, reinvest_rate):
...
+        --------
+        rf = 0.07, beta = 12, rm = 0.10
+        
+        min.required rate of return = 0.07 + 1.2(0.10 - 0.07)
+                                    =0.106
+                                    =10.6%
+
+    """
+    try:
+        rf >= 1 or rm >= 1:
```
Comment (eric-wieser) [inline numpy/lib/financial.py]: SyntaxError here.

File: numpy/lib/financial.py [hunk 2]
```diff
@@ -759,3 +759,51 @@ def mirr(values, finance_rate, reinvest_rate):
...
+            return "values cannot be greater than or equals 1"
+        else:
+            ke = rf + beta * (rm - rf)
+            return ke
+    except Exception as e:
+        raise e
```
Comment (eric-wieser) [inline numpy/lib/financial.py]: This `try`/`except` can be removed

**Review by eric-wieser — COMMENTED** [review]
I think there's a general reluctance to add new financial functions to numpy: #2880.

File: numpy/lib/financial.py (hunk 1 above)
Comment (olamilekan000) [reply numpy/lib/financial.py]: oops! I missed out the if statement there.

Comment (olamilekan000) [reply numpy/lib/financial.py]: "I think there's a general reluctance to add new financial functions to numpy: #2880."
Does it mean contributing to the finance library is void for now?

Comment (rgommers) [reply numpy/lib/financial.py]: @olamilekan000 indeed. I've added a comment to gh-2880 to that effect.

**Comment by rgommers** [comment]
@olamilekan000 thank you for contributing. we don't want to accept new financial functions, hence I'm closing this PR. We do appreciate your efforts though.

**Pull request closed without merging**